apps/fluent-tester/src/FluentTester/TestComponents/Persona/StandardUsage.tsx

```diff
@@ -43,6 +45,34 @@ export const StandardUsage: React.FunctionComponent = () => {
   const [showTertiary, setShowTertiary] = React.useState(true);
   const [showOptional, setShowOptional] = React.useState(true);
   const [imageSize, setImageSize] = React.useState<PersonaSize | undefined>('size72');
+  const renderMenuPicker = Platform.OS == ('win32' as any) || Platform.OS == 'macos';
+
+  const Pickers = () => {
+    return (
+      <Picker
```

Comment:
Seems a little odd to me that we write two sets of UI for each test that uses pickers. I think it'd be better to have a component which wraps MenuPicker and Picker and renders one or the other based on the platform. That way we're not duplicating picker code for each test with pickers.

Reply:
Also with the way it is written right now, I think the way it works out is that both sets of code run on all platforms and then only one of the two gets used.